insights/specs/default.py

```diff
@@ -418,6 +418,7 @@ def semid(broker):
     ls_etc = simple_command("/bin/ls -lanR /etc")
     ls_lib_firmware = simple_command("/bin/ls -lanR /lib/firmware")
     ls_ocp_cni_openshift_sdn = simple_command("/bin/ls -l /var/lib/cni/networks/openshift-sdn")
+    ls_origin_local_volumes_pods = listdir("/var/lib/origin/openshift.local.volumes/pods")
```

Comment:
@csams - Would you please to have a confirm on this? if it's OK to use the `listdir` as a datasource directly? (instead of using the DatasourceProvider) It's tested OK on RHEL host, but I'm not sure if it works well with the client host.

Reply:
@xiangce the `listdir` spec is a datasource (see [__init__()](https://github.com/RedHatInsights/insights-core/blob/master/insights/core/spec_factory.py#L664)) that returns a list of names with no attributes, so if that's all the parser needs then it should work.

Reply:
@bfahr I updated the code according to your comments. I choose to use `simple_command("ls -l /var/lib/origin/openshift.local.volumes/pods")` instead, because it contains more information (OpenShift pods created time).